.ci/flake8_wrapper_docstrings.sh

```diff
@@ -7,7 +7,7 @@ set -e
 # D3XX - Quoting issues.
 # D401 - First line should be in imperative mood 
 # D403 - First word of the first line should be properly capitalized
-args="--ignore=D --select=D100,D201,D202,D206,D207,D208,D209,D211,D3,D401,D403"
+args="--ignore=D --select=D100,D201,D202,D206,D207,D208,D209,D211,D3,D403"
```

Comment:
By looking at the above comments, it seems that actually D401 was supposed to be enforced on the files included in `flake8_docstrings_include_list.txt` . Maybe remove `lib/galaxy/containers` from it instead?